objects.md

```diff
@@ -0,0 +1,118 @@
+# About
+
+This repo needs to represent the following information:
+
+1. Adding/removing versions to channels
+2. Remove or add edges to all channels
+3. Remove or add edges to a specific channel
+4. Implement an "upgrade policy" object specifically for y-streams
+
+Remember that cininnatti should pull most edges from the release manifest. This data should typiclly remove edges, not add them.
+
+# Directory Structure
+
+We do this via a combination of objects and directory structure. All object should be YAML (so we can make comments). The directories in the repo should be:
+- /nodes/
+- /channels/
+- /added_edges/
+- /blocked_edges/
+- /y-policies/
+
+# Nodes
+
+The /nodes objects are very simple. It is not strictly necessary as long as cincinnatti pulls all releases from quay. However it is useful for a CI tool to build a final (smaller) graph and for disconnected customers to build a complete cincinnatti graph. (though the later is not the first target goal!) So /nodes/4.1.24 would look like:
+```
+version: 4.1.24
+payload: quay.io/openshift-release-dev/ocp-release@sha256:6f87fb66dfa907db03981e69474ea3069deab66358c18d965f6331bd727ff23f
```

Comment:
This saves us a tag-listing query, but we still have to hit the registry to get metadata, etc.  Doesn't seem to be worth the trouble.  I'd drop it unless it it grows to look like #1 and include all node metadata (so Cincinnati doesn't have to hit the registry at all).

Reply:
I believe that for 'disconnected' people this format would allow someone to generate a complete graph (even without the in payload metadata)

I completely agree, not strictly required to start.